lib/rectangles.js

```diff
@@ -1,17 +1,17 @@
-function area(l, w) {
-  return l * w;
+function area(rectangle) {
+  return rectangle.length * rectangle.width;
```

Comment:
A Rectangle class would have been useful. On the Triangle class I ran into an issue where I had to put the helper functions outside of the class definition, or else they came back undefined when called from within the class. That doesn't seem like the right way to do it, though. I must be missing something regarding `self` or `this` or something, and when I figure it out I'll refactor to use classes properly.